### PR TITLE
Warcraft logs uploader

### PR DESCRIPTION
**Pull request by phschott**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by p-linnane** [comment]
Hi. Thanks for the PR. We have notability requirements for inclusion in homebrew-cask (<30 forks, <30 watchers and <75 stars), which unfortunately this software does not meet. Please consider hosting this in [your own Homebrew tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) instead.

**Comment by bevanjkay** [comment]
@p-linnane I don't think the notability requirement applies here because the repository only hosts the binaries, not the source code.

File: Casks/w/warcraft-logs-uploader.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  url "https://github.com/RPGLogs/Uploaders-warcraftlogs/releases/download/v#{version}/warcraftlogs-v#{version}.dmg",
+      verified: "github.com/RPGLogs/Uploaders-warcraftlogs/"
+  name "Warcraft logs"
+  desc "Client to upload warcraft logs"
+  homepage "https://classic.warcraftlogs.com/"
+
+  livecheck do
+    url "https://classic.warcraftlogs.com/client/download"
+    strategy :page_match
+    regex(%r{.*?/warcraftlogs[._-]v?(\d+(?:\.\d+)*)\.dmg}i)
```
Comment (miccal) [inline Casks/w/warcraft-logs-uploader.rb:14]:
```suggestion
    regex(%r{.*?/warcraftlogs[._-]v?(\d+(?:\.\d+)+)\.dmg}i)
```
The `:page_match` strategy is the default strategy, so it does not need to be stated here.

Comment (miccal) [inline Casks/w/warcraft-logs-uploader.rb:14]:
```suggestion
    regex(%r{.*?/warcraftlogs[._-]v?(\d+(?:\.\d+)+)\.dmg}i)
```
The `:page_match` strategy is the default `livecheck` strategy, to this line can be removed.

**Comment by miccal** [comment]
Thank you @phschott.

**Review by miccal — APPROVED** [review]

**Pull request merged**